src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import NavBar from './components/NavBar';
+// import NavBar from './components/NavBar';
```

Comment:
```suggestion
import NavBar from './components/NavBar';
```
You could leave the changes here, and just add your component "Rocket" to the existing code in the dev branch

Reply:
okay I will after I merge thank you.